src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/layout.html.twig

```diff
@@ -39,7 +39,7 @@
                                     {%- else -%}
                                             {{ redirect_route }}
                                     {%- endif %}
-                                    (<a href="{{ path('_profiler', { token: redirect.token }) }}">{{ redirect.token }}</a>)
+                                    (<a href="{{ path('_profiler', { token: redirect.token, panel: app.request.query.get('panel', 'request') }) }}">{{ redirect.token }}</a>)
```

Comment:
In case it's null I'd prefer to set the default panel name instead of relying of auto-redirection to some panel. But your proposal is correct too. They are just different opinions on the same subject!